packages/zoe/src/types.js

```diff
@@ -8,17 +8,16 @@
 
 /**
  * @template {string} H - the name of the handle
- * @typedef {H & {}} Handle A type constructor for an opaque type identified by the H string.
- * This uses an intersection type ('MyHandle' & {}) to tag the handle's type even though the
+ * @typedef {H & {}} Handle A type constructor for an opaque type identified by
+ * the H string.
+ * This uses an intersection type ('MyHandle' & {}) to tag the handle's type even
+ * though the
  * actual value is just an empty object.
  */
 
 /**
  * @typedef {string} Keyword
- * @typedef {Handle<'InstallationHandle'>} InstallationHandle - an opaque handle for an bundle installation
- * @typedef {Handle<'InstanceHandle'>} InstanceHandle - an opaque handle for a contract instance
- * @typedef {Handle<'OfferHandle'>} OfferHandle - an opaque handle for an offer
- * @typedef {Handle<'InviteHandle'>} InviteHandle - an opaque handle for an invite
+ * @typedef {Handle<'InvitationHandle'>} InvitationHandle - an opaque handle for an invite
```

Comment:
```suggestion
 * @typedef {Handle<'InvitationHandle'>} InvitationHandle - an opaque handle for an invitation
```

Is it still opaque, or is it growing any methods?

Reply:
Still opaque, has not changed